.coderabbit.yaml

```
@@ -0,0 +1,28 @@
+        2. Cryptographic operations and key handling
+        3. Error handling and propagation
+        4. PyO3 usage and Python integration
+        5. API design and documentation
+
+    - path: "**/*.pyi"
+      instructions: |
+        These are Python files stubs for the Rust bindings.
+        Pay special attention to:
+        1. Type safety and proper type hints
+        2. API usage examples
+        3. Documentation
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)